flows/waits/base.go

```diff
@@ -0,0 +1,27 @@
+package waits
+
+import (
+	"time"
+
+	"github.com/nyaruka/goflow/flows"
+)
+
+type BaseWait struct {
```

Comment:
Yeah we could do that - stops users from putting a timeout on a nothing wait.

Waits in the flow definition can have a "timeout" value and session can checked on a wait to see if it "has timed out". I think this is clear, no?